benches/misc/coprocessor/codec/batch/mod.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use test::Bencher;
+use tikv::coprocessor::codec::batch::*;
+use tikv::coprocessor::codec::mysql::*;
+
+#[bench]
+fn bench_column_agg_long(b: &mut Bencher) {
```

Comment:
Please also describe what this benchmark is testing.. Looks like actually it tests multiple things together? Currently I don't get it because for decimals looks like Decimal adding and constructing is also included so that the time of benchmark is not purely the time of accessing elements?